tests/ConfigTest.php

```diff
@@ -107,7 +107,7 @@ public function testThatFinderWorksWithDirSetOnConfig()
         $config = new Config();
 
         $iterator = $config->getFinder()->in(__DIR__.'/Fixtures/FinderDirectory')->getIterator();
-        $this->assertSame(1, count($iterator));
+        $this->assertSame(1, iterator_count($iterator));
```

Comment:
interesting, do you have an ref for that change ?
would `assertCount` finally works ?